### ISSUE TITLE
Events arrive in Sentry, but not in Bugsink

### ISSUE THREAD

**Issue by jeriox** [bugsink/bugsink#11]
Hey there,

we are currently evaluating Bugsink as an alternative solution to bugsink. While the installation was a breeze, we cannot get our python-django project to properly send events to our bugsink. When using a Sentry DSN, the events arrive there without a problem. When swapping the DSN for the one displayed by bugsink, nothing happens (no events appear in bugsink). What is the best way to troubleshoot this issue?

**Comment by vanschelven** [comment]
Fair point, I'll look into a nice way of doing this.

Hope the evaluation goes well; you can always jump on [discord](https://discord.gg/AEgr6Bp8) if you run into further issues.

**Comment by jeriox** [comment]
Thanks!

that discord invite is expired btw

**Comment by vanschelven** [comment]
> expired

thanks, should be fixed now!

**Comment by vanschelven** [comment]
> A quick reference that snappea also reads the bugsink.conf would be helpful, it took a moment to understand that snappea needed a restart after changes there as well.

the [settings documentation](https://www.bugsink.com/docs/settings/) already contains the following note, but I understand that a note like that is not necessarily top-of-mind when you're trying to figure out why it doesn't work.

> Changing settings in `bugsink_conf.py` will require a **restart of the webserver** and (if you're running that)
**the snappea worker**. Changing settings using environment variables will require a restart of the webserver.

I'm certainly open to more/better locations though!

**Comment by jeriox** [comment]
yeah that was the main reason for the issue, giving a path to the config-creation tool would have made things a lot easier (and prevented all the other errors). The restart was just a sidenote, I figured that out quite quickly :)
